iOS Example/iOS Example/ViewController.swift

```diff
@@ -2,16 +2,14 @@ import Authgear
 import UIKit
 
 class ViewController: UIViewController {
-    let container = Authgear(clientId: "client_id", endpoint: "http://localhost:3000")
+    var container: Authgear?
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-
-        container.configure()
+    public func configure(container: Authgear) {
+        self.container = container
```

Comment:
Remove the property from this view controller. Instead, add a computed property to get the container from AppDelegate. Also remove this configure function.

something like

```swift
private(get) var container: Authgear? {
  let appDelegate = UIApplication.shared.delegate as? AppDelegate
  return appDelegate.container
}

func login(_ sender: Any) {
  self.container?.authorize(...)
}
```